packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -363,6 +408,101 @@ contract OPContractsManager is ISemver {
         return output;
     }
 
+    function addGameType(AddGameInput[] memory _gameConfigs) external returns (AddGameOutput[] memory) {
+        if (address(this) == address(thisOPCM)) revert OnlyDelegatecall();
+
+        AddGameOutput[] memory outputs = new AddGameOutput[](_gameConfigs.length);
+        Blueprints memory bps = thisOPCM.blueprints();
+
+        for (uint256 i = 0; i < _gameConfigs.length; i++) {
+            AddGameInput memory gameConfig = _gameConfigs[i];
+
+            IPermissionedDisputeGame pdg = IPermissionedDisputeGame(
+                address(
+                    IDisputeGameFactory(gameConfig.systemConfig.disputeGameFactory()).gameImpls(
+                        GameTypes.PERMISSIONED_CANNON
+                    )
+                )
+            );
+            uint256 l2ChainId = pdg.l2ChainId();
+
+            outputs[i].delayedWETH = IDelayedWETH(
+                payable(deployProxy(l2ChainId, gameConfig.proxyAdmin, gameConfig.saltMixer, "DelayedWETH"))
```

Comment:
Should the `proxyAdmin` value be pulled from some source-of-truth for this chain rather than allowing it to be configurable?

Reply:
correct. we can't get the proxyadmin though, so the user has to specify it.